modules/installation-gcp-enabling-confidential-vms.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+// * installing/installing_gcp/installing-gcp-network-customizations.adoc
+// * installing/installing_gcp/installing-gcp-private.adoc
+// * installing/installing_gcp/installing-gcp-vpc.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.adoc
+// * installing/installing_gcp/installing-gcp-user-infra.adoc
+// * installing/installing_gcp/installing-gcp-user-infra-vpc.adoc
+// * installing/installing_gcp/installing-restricted-networks-gcp.adoc
+
+:_content-type: PROCEDURE
+[id="installation-gcp-enabling-confidential-vms_{context}"]
+= Enabling Confidential VMs
+In {product-title} {product-version}, you can use Confidential VMs when installing your cluster. Confidential VMs encrypt data while it is being processed. For more information, see Google's documentation on link:https://cloud.google.com/confidential-computing[Confidential Computing]. You can enable Confidential VMs and Shielded VMs at the same time, although they are not dependent on each other.
+
+.Prerequisites
+* You have created an `install-config.yaml` file.
+
+.Procedure
+
+* Use a text editor to edit the `install-config.yaml` file prior to deploying your cluster and add one of the following stanzas:
+.. To use confidential VMs for only control plane machines:
++
+[source,yaml]
+----
+controlPlane:
+  platform:
+    gcp:
+       confidentialCompute: Enabled <1>
+       type: n2d-standard-8 <2>
+       onHostMaintenance: Terminate <3>
+----
+<1> Enable confidential VMs. 
+<2> Specify a machine type that supports Confidential VMs. Confidential VMs require the N2D or C2D series of machine types.
```

Comment:
You can refer to this link
https://cloud.google.com/compute/confidential-vm/docs/os-and-machine-type#machine-type